express-model/templates/_.server.model.tests.js

```diff
@@ -11,7 +11,8 @@ var should = require('should'),
 /**
  * Globals
  */
-var user, <%= camelizedModelName %>;
+var user,
```

Comment:
Under mean.js 0.5beta, spacing is enforced:
*...modules/pickup/tests/server/example.server.model.tests.js*
`14:11  error  Expected variable declaration to be on a new line  one-var-declaration-per-line`
`30:27  error  Trailing spaces not allowed                        no-trailing-spaces`
 `49:29  error  Trailing spaces not allowed `